src/main/java/intellistart/interviewplanning/controllers/dto/DashboardDto.kt

```diff
@@ -0,0 +1,10 @@
+package intellistart.interviewplanning.controllers.dto
+
+/**
+ * Dto object for representation of all slots and bookings per certain time period.
+ */
+data class DashboardDto (
+    val interviewerSlots: MutableSet<DashboardInterviewerSlotDto>,
```

Comment:
should these collections be mutable? by default use immutable collections, mutable - only when there's reason for it

Reply:
These collections are populated later in the addInterviewerSlots and addCandidateSlots methods in the DashboardMapDto class and use the add(...) method internally. If these collections are not mutable, then this method does not work